src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,48 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber {
+    private final int number;
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
```

Comment:
상수를 인스턴스 변수보다 상위에 위치시켜주세요. 

```suggestion
    private static final int MIN_LOTTO_NUMBER = 1;
    private static final int MAX_LOTTO_NUMBER = 45;

    private final int number;
```

Reply:
반영했습니다!